com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/helpers/ApplicationColorDetector.java

```diff
@@ -104,7 +103,7 @@ private ApplicationColor getApplicationColor(DeployedMtaModule deployedMtaModule
                      .filter(color -> deployedMtaModule.getAppName()
                                                        .endsWith(color.asSuffix()))
                      .findFirst()
-                     .orElse(COLOR_OF_APPLICATIONS_WITHOUT_SUFFIX);
+                     .orElse(DeployedLiveModuleCalculator.COLOR_OF_APPLICATIONS_WITHOUT_SUFFIX);
```

Comment:
I think this constant is better here, rather than in the calculator

Reply:
The reason for moving is that DeployedModuleCalculator doesn't have dependency to cf.process package and can't access classes from cf.processes, so the constant should be moved somewhere where it could be accessible from cf.core and cf.processes packages.